docs/changelog.rst

```diff
@@ -16,6 +16,7 @@ Changelog
 * :bug:`2628` Users will now see the correct total asset value when visiting an asset's detail page for a second time.
 * :bug:`2524` Users will now not be able to delete assets from the Global DB if any account in the local system owns them.
 * :bug:`2631` Balancer trades will now be displaying the correct trade rate, both in the trade history section but also in the exported CSV.
+* :bug:`2633` Deposits and withdrawals that have not been completed in Coinbase Pro will not raise an exception at the moment of being processed.
```

Comment:
nitpick on the changelog. Make is sound more user-friendly. Users don't know what an exception is.
```suggestion
* :bug:`2633` User with incomplete deposits and withdrawals in Coinbase Pro will now be able to generate a profit/loss report.
```